# Common
policy_update_rate: 0.0  # execute as fast as possible
reference_update_rate: 10.0
publish_ros: true
ros_publish_rate: 10.0

gaussian_policy: true
sequential: false

default_pose: [0.15, -0.05, -0.02, -1.87, 0.06, 1.79, 0.0]
#default_pose: [1, 1, 1, 1, 1, 1, 1]
object_tolerance: 0.09 # ~5 deg
solver:
  rollouts: 40
  lambda: 0.0001 #0.001
  h: 10.0
  step_size: 0.015
  horizon: 0.5

  cost_ratio: 0.2
  caching_factor: 0.3
  substeps: 1

  gradient_step_size: 1.0
  momentum_step_size: 0.0

  filtering: true
  filters_order: [1, 1, 1, 1, 1, 1, 1, 1, 1]
  filters_window: [10, 10, 10, 10, 10, 10, 10 ,10, 10]

  bound_input: true
  u_min: [-1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0]
  u_max: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0 , 1.0]

  spline_degree: 3
  spline_dt: 0.1
  spline_step_size: 0.1

  adaptive_sampling: false
  # input_variance: [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25]
  input_variance: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5]

  discount_factor: 1.0
  verbose: false
  threads: 8
  debug_print: false

  logging: false
